squire/urls.py

```diff
@@ -45,6 +45,9 @@
     path('', include('core.urls', namespace='core')),
 
     path('', include('user_interaction.urls')),
+
+    # Shortcuts, always at the end
```

Comment:
maybe clarify why it should be at the end? (i guess shortcuts could overtake existing pages otherwise?)